src/components/AddLinkInput/AddLinkInput.tsx

```diff
@@ -0,0 +1,56 @@
+import { CSSProperties, useEffect, useRef, useState } from "react";
+import styles from "./AddLinkInput.module.scss";
+import InputIcon from "@/assets/svg/InputIcon";
+
+export default function AddLinkInput() {
+  const [isAddLinkInputVisible, setAddLinkInputVisible] = useState(true);
```

Comment:
이렇게 되어야 컨벤션에 맞을 것 같네요~
```suggestion
  const [isAddLinkInputVisible, setIsAddLinkInputVisible] = useState(true);
```

Reply:
이런 디테일한 컨벤션까지... 너무 감사합니다 ㅠㅠ 바로 수정하겠습니다!